app/controllers/prx_auth/rails/sessions_controller.rb

```diff
@@ -48,13 +48,13 @@ def auth_error
     end
 
     def create
+      clear_nonce!
```

Comment:
Err... isn't this going to break things?  Clearing the nonce before checking if it's valid.

Reply:
Yeah, confirmed this is breaking.

(We should really add the tests for these repos to github actions...)

Reply:
Okay, i fixed it!